developer_manual/app/backgroundjobs.rst

```diff
@@ -4,14 +4,22 @@ Background Jobs (Cron)
 
 .. sectionauthor:: Bernhard Posselt <dev@bernhard-posselt.com>
 
-Background/cron jobs are usually registered in the :file:`appinfo/app.php` by using the **addRegularTask** method, the class and the method to run:
+Background/cron jobs should be registered in the :file:`appinfo/install.php` and :file:`appinfo/update.php`. Previously this was mostly done in :file:`appinfo/app.php`, but that causes a query on each page load.
```

Comment:
It's there for the case the app existed before you added the background job. For a completely new app the install.php would be enough. But for an existing app this needs to be in both.